### ISSUE TITLE
Initial value does not get set on remote typeahead when multiple remote typeaheads are displayed 

### ISSUE THREAD

**Issue by m4rc0z** [baloise-incubator/design-system#603]
The setting of a initial value of the remote typeaheads do not work when multiple remote typeaheads are displayed on one page. 

I provided an example branch with the name (feat/remote_typeahead_initialization_bug) in this [repository](https://github.com/baloise/design-system-examples).

On initial run of the application sometimes it works. But when you do a reload of the page it does not work anymore.

Hint:

I did a small analysis and could be that this function is the reason for the issue:

```
 updateInputValue(value) {
    if (this.updateInputValueTimer) {
      clearTimeout(this.updateInputValueTimer);
    }
    this.updateInputValueTimer = setTimeout(() => {
      if (!lodash_isnil(this.inputElement)) {
        this.inputElement.value = value;
        this.inputValue = value;
      }
    }, 0);
  }

```
since it uses timeoutes maybe it is a timing issue.

**Comment by joachimprinzbach** [comment]
Will investigate this one, roughly mid of next week

**Comment by hirsch88** [comment]
@nobilo & @m4rc0z could you retest that with the newest version?

**Comment by m4rc0z** [comment]
if i have a look at the created branch version with the usage of the new design system version 10.24.9 the values are still not initialized.

**Comment by hirsch88** [comment]
@m4rc0z can u test this version @baloise/design-system-components-angular@11.0.3-beta.2

**Comment by tklueber** [comment]
Super thanks - it looks good to me!

I have reproduced the issue in the Example app and it seems to be fixed with the 11.0.3-beta.2 ([see branch 603/initial_value_typeahead](https://github.com/baloise/design-system-examples/tree/603/initial_value_typeahead))

(but finally, I would try the fix again in our real application, and check if it works there too).
